content/v2.0/reference/glossary.md

```diff
@@ -90,6 +90,8 @@ Related entries: [implicit block](#implicit-block), [explicit block](#explicit-b
 
 A data type with two possible values: true or false.
 By convention, you can express `true` as the integer `1` and false as the integer `0` (zero).
+In [annotated CSV](/v2.0/reference/syntax/annotated-csv/), columns that contain
```

Comment:
Does "in annotated CSV" refer to both csv2lp library annotations and the "standard" annotated CSV? If not, is there any notable distinction?